package.json

```diff
@@ -9,7 +9,15 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "postinstall": "touch node_modules",
+    "pretest": "npm install",
+    "test": "_mocha",
```

Comment:
The following will fix `0.12` in the short term which we need. We're moving to Babel so this won't be needed for long.
    "node_flags": "node --v8-options | grep generators | cut -d ' ' -f 3",
    "test": "node `npm run -s node_flags` ./node_modules/.bin/_mocha"

_Note_: This won't work for Windows but `npm run test-debug` will work in the short term if needed.